src/coreclr/vm/appdomain.cpp

```diff
@@ -1589,6 +1589,9 @@ Module* SystemDomain::GetCallersModule(StackCrawlMark* stackMark)
     }
     CONTRACTL_END;
 
+    if (stackMark == NULL)
```

Comment:
Why would this ever be true?

Reply:
Some of the new callers that I have added are passing NULL for the stackmark.